t-sql/t-sql-quiz.md

```diff
@@ -611,3 +611,22 @@ SELECT 123+'abc' AS Result;
 - [ ] 123'abc'
 - [ ] '123abc'
 - [ ] error
+
+
+#### Q50.What output will the following SQL sequence produce? Assume that the tables have been created and all the columns exist.
+
+```tsql
+INSERT INTO Account (acct,bal) VALUES ('12345', 100);
+UPDATE Account SET bal=bal+100;
+BEGIN;
+UPDATE Account SET bal=bal+100.
+ROLLBACK;
+SELECT bal FROM Account WHERE acct='12345';
+);
+```
+
+- [ ] 100
+- [x] 200
+- [ ] 300
+- [ ] `You will get an error because ROLLBACK deletes the row that was update`
+[Reference link](https://www.geeksforgeeks.org/sql-transactions/)
```

Comment:
```suggestion
- [ ] `You will get an error because ROLLBACK deletes the row that was update`

[Reference link](https://www.geeksforgeeks.org/sql-transactions/)
```